docs/pages/index.mdx

```diff
@@ -83,7 +83,7 @@ And verification is as simple as running:
 cargo nexus verify
 ```
 
-To get started with the Nexus zkVM, check out the [Quick Start](zkvm/quick-start.mdx) guide.
+To get started with the Nexus zkVM, check out the [SDK Quick Start](zkvm/sdk-quick-start.mdx) guide.
```

Comment:
Maybe link to both?

`check out the SDK Quick Start or CLI Quick Start`?

Reply:
makes sense, I'm on it

Reply:
done